Bindings/Packages/com.virtualmaker.bindings/Runtime/VirtualMaker.Bindings.asmdef

```diff
@@ -11,6 +13,42 @@
     "precompiledReferences": [],
     "autoReferenced": true,
     "defineConstraints": [],
-    "versionDefines": [],
+    "versionDefines": [
+        {
+            "name": "com.unity.ugui",
+            "expression": "",
+            "define": "BINDINGS_UGUI"
+        },
+        {
+            "name": "com.unity.modules.uielements",
+            "expression": "",
+            "define": "BINDINGS_UI_ELEMENTS"
+        },
+        {
+            "name": "com.unity.modules.unitywebrequesttexture",
+            "expression": "",
+            "define": "BINDINGS_WEB_REQUEST_TEXTURE"
+        },
+        {
+            "name": "com.unity.modules.unitywebrequest",
```

Comment:
do you need this one if you have the one above?

Reply:
Agreed, pretty sure it is direct dep